models/src/main/protobuf/DeployServiceV1.proto

```diff
@@ -34,6 +34,7 @@ option (scalapb.options) = {
 service DeployService {
   // Queue deployment of Rholang code (or fail to parse).
   rpc doDeploy(DeployDataProto) returns (DeployResponse) {}
+  rpc deployStatus(DeployStatusQuery) returns (DeployStatusResponse) {}
```

Comment:
Please add short description in comments.

Reply:
[Done](https://github.com/rchain/rchain/pull/3695/commits/d035a9ce61b501cbcea69ce579f1d24435d35c52).